### ISSUE TITLE
precisedelta is not included in pypi package

### ISSUE THREAD

**Issue by Naught0** [jmoiron/humanize#149]
### What did you do?
Installed humanize & attempted to use `precisedelta` function

### What did you expect to happen?
For the function to exist in the package

### What actually happened?
The function does not exist in the time.py module & fails to import

### What versions are you using?

* OS: Win 10 
* Python: 3.7.4
* Humanize: 2.5.0

Please include **code** that reproduces the issue.

```python
>>> import humanize
>>> import datetime as dt
>>> humanize.time.precisedelta(dt.date.today() - dt.date(2020,7,1))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'humanize.time' has no attribute 'precisedelta'
>>> humanize.precisedelta(dt.date.today() - dt.date(2020,7,1))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'humanize' has no attribute 'precisedelta'
```

![](https://i.imgur.com/Ku4Pnmh.png)

**Comment by hugovk** [comment]
`precisedelta` was added in https://github.com/jmoiron/humanize/pull/137 but hasn't been released yet.

I'll make a release soon, in the meantime you can install from `master`:

pip install -U git+https://github.com/jmoiron/humanize

**Comment by Naught0** [comment]
Thanks! I saw it was included in the Github code, but not PyPi & it works like a charm now that I've used the proper package.

**Comment by hugovk** [comment]
And I've just released 2.6.0 as well!

* https://github.com/jmoiron/humanize/releases/tag/2.6.0
* https://pypi.org/project/humanize/2.6.0/

**Comment by Naught0** [comment]
I coincidentally updated to the latest version seconds ago and got that version! 🎉 ❤️
